pages/ecosystem.vue

```diff
@@ -136,177 +137,141 @@ useHead({
 });
 
-// TODO: Integrate old code
-// const { data: members } = useLazyAsyncData(
-//   "fetch-members",
-//   async () => await import("~/content/ecosystem/members.json")
-// );
-
-// const { data: tiers } = useLazyAsyncData(
-//   "fetch-tiers",
-//   async () => await import("~/content/ecosystem/tiers.json")
-// );
+const tierFilters = ref<string[]>([]);
 
-// const tierFilters = ref<string[]>([]);
+const tierFiltersAsString = computed(() => tierFilters.value.join(","));
 
-// const noTierFiltersSelected = computed(() => tierFilters.value.length === 0);
+const filteredMembers = computed(() => {
+  if (!members) {
+    return [];
+  }
 
-// const filteredMembers = computed(() => {
-//   return noTierFiltersSelected
-//     ? members.value
-//     : members.value.filter((member) => tierFilters.value.includes(member.tier));
-// });
+  const noTierFilters = tierFilters.value.length === 0;
 
-// const tiersNames = computed(() => tiers.value.map((tier: any) => tier.name));
-
-// function getTestRows(member: any) {
-//   if (member.testsResults) {
-//     return member.testsResults.map((res: any) => {
-//       // make tiemstamp human readable
-//       const timestamp = new Date(res.timestamp * 1000).toLocaleString("en-UK", {
-//         timeZone: "UTC",
-//       });
-//       // Convert package name to title case
-//       let packageName;
-//       if (res.package) {
-//         packageName = res.package
-//           .replaceAll("-", " ")
-//           .split(" ")
-//           .map((s: string) => s.charAt(0).toUpperCase() + s.substring(1))
-//           .join(" ");
-//       }
-
-//       return {
-//         packageName,
-//         packageVersion: res.packageVersion,
-//         pythonVersion: res.pythonVersion,
-//         qiskitVersion: res.qiskitVersion,
-//         status: res.status,
-//         timestamp,
-//       };
-//     });
-//   }
-// }
-
-// function getTierDescription(tierName: string): string {
-//   const tier = tiers.value.find((tier: any) => tier.name === tierName);
-//   return tier.description || "";
-// }
-
-// function updateTierFilter(filterValue: string, isChecked: boolean) {
-//   isChecked
-//     ? tierFilters.value.push(filterValue)
-//     : (tierFilters.value = tierFilters.value.filter(
-//         (tier: any) => tier !== filterValue
-//       ));
-// }
-
-// const updateTierFilters = (filterValues: string[]) => {
-//   tierFilters.value = filterValues;
-// };
+  return noTierFilters
+    ? members
+    : members.filter((member) => tierFilters.value.includes(member.tier));
+});
 
-// function isTierFilterChecked(filterValue: string): boolean {
-//   return tierFilters.value.includes(filterValue);
-// }
+const tiersNames = computed(() => tiers.map((tier) => tier.name));
+
+function getTestRows(member: Member) {
+  if (member.testsResults) {
+    return member.testsResults.map((res) => {
+      // make timestamp human readable
+      const timestamp = new Date(res.timestamp * 1000).toLocaleString("en-UK", {
```

Comment:
This code is repeated on the `<template>`. Could you create a function and reuse it?

Reply:
Done in https://github.com/Qiskit/qiskit.org/pull/3001/commits/59db4848bd17e454eb65ffd4ce81827f3dceeb13.